src/App.jsx

```diff
@@ -3,16 +3,18 @@ import { Route, Routes, BrowserRouter as Router } from 'react-router-dom';
 import { CookiesProvider } from 'react-cookie';
 import Login from './components/Login/Login';
 import Logout from './components/Logout';
+import Layout from './components/Layout/Layout';
+import LoginPage from './pages/LoginPage/LoginPage';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module './pages/LoginPage/LoginPage'.